templates/workflows/security_npm_dependency.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   security-npm-dependency-check:
     name: Project security npm dependency check
-    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_npm_dependency.yml@v0.7.1 # WORKFLOW_VERSION
+    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_npm_dependency.yml@v1.0 # WORKFLOW_VERSION
```

Comment:
should the templates actually be `@v1` instead?  That way they pin to a major version so will allow minor and patch updates

Reply:
Makes good sense - I'll double-tag so it'll be v1.0 and v1
Just a mo!